coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java

```diff
@@ -49,7 +49,7 @@ public void beforeClass() throws HiveException, MetaException {
     UnsupportedHiveUDFsInSpark.add("com.linkedin.coral.hive.hive2rel.CoralTestUnsupportedUDF");
 
     TransportableUDFMap.add("com.linkedin.coral.hive.hive2rel.CoralTestUDF", "com.linkedin.coral.spark.CoralTestUDF",
-        "ivy://com.linkedin.coral.spark.CoralTestUDF");
+        "ivy://com.linkedin.coral.spark.CoralTestUDF", null);
```

Comment:
Can we have a test for Spark 3? It can fail if needed.

Reply:
Added integration tests.